input/pagecontent/overview.md

```diff
@@ -424,25 +423,24 @@ In this section, we will walk through a realistic example of FSH, line by line.
 A few things to note about this example:
 
 * The order of the items (aliases, profile, value set, extension) doesn't matter. In FSH, you can refer to items defined before or after the current item. By convention, aliases appear at the beginning of a file.
-* The example assumes the items are all in one file, but they could be in separate files. The allocation of items to files is the author's choice.
+* The example assumes the items are all in one file, but they could be in separate files. The allocation of items to files is the author's choice. Aliases defined in one file can be referred to in other files.
 * Most of the rules refer to elements by their FHIR names, but when the rule refers to an element that is not at the top level, more complex paths are required. An example of a complex path occurs on line 10, `extension[evidenceType].valueCodeableConcept`. The Language Reference contains [further descriptions of paths](reference.html#fsh-paths).
 
 ### Future Considerations
 
 In this introduction, we presented an overview of FSH. Not all features were covered. A complete accounting of the language is found in the [FSH Language Reference](reference.html).
 
-While this version of FSH is capable of producing complex IGs, future versions may introduce additional features. Feature suggestions are welcome, and can be made [in the HL7 Jira](https://jira.hl7.org/browse/FHIR-27321?jql=project%3D%22FHIR%22%20AND%20Specification%20%3D%20%22Shorthand%20(FHIR)%20%5BFHIR-shorthand%5D%22%20). 
+While this version of FSH is capable of producing sophisticated IGs, future versions may introduce additional features. FSH language suggestions are welcome, and can be made [in the HL7 Jira](https://jira.hl7.org/browse/FHIR-27321?jql=project%3D%22FHIR%22%20AND%20Specification%20%3D%20%22Shorthand%20(FHIR)%20%5BFHIR-shorthand%5D%22%20). SUSHI issues and suggestions can be made [here](https://github.com/FHIR/sushi/issues).
 
-> **Note:** When filing issues, use project="FHIR" AND Specification = "Shorthand (FHIR) [FHIR-shorthand]".
+> **Note:** When filing FSH language or IG issues, use project="FHIR" AND Specification = "Shorthand (FHIR) [FHIR-shorthand]".
 
 Some of the features for FSH under consideration include (in no particular order):
 
-* **Slicing Support:** Currently, slicing requires the user to specify discriminator type, path, and slicing rules. A future version of SUSHI may handle slicing situations without explicit declarations by the user. However, common slicing patterns now can be captured in RuleSets, relieving the need for this feature.
+* **Improved Slicing Support:** Currently, slicing requires the user to specify discriminator type, path, and slicing rules. A future version of SUSHI may handle slicing situations without explicit declarations by the user. However, common slicing patterns now can be captured in RuleSets, lessening the need for this feature.
```

Comment:
You note the applicability of RuleSets here; it probably makes sense to also modify the section below about conformance resources to also indicate that parameterized RuleSets may be a viable option rather than purpose-specific syntax.

Reply:
I actually think this section is of minimal value. I removed it and I don't think anything is lost. Some of the content worth retaining, such as the CapabilityStatementTemplate has been moved.

Reply:
Moreover, the example like the Capability Statement template is better and more consistent included in FSH Online examples. The Template involved a complex profile, and did not need that. As you show in  your Advanced FSH Tutorial, rules are effective. Added example to FSH-Online-Examples (PR in review).